src/common/snippets/include/snippets/op/loop.hpp

```diff
@@ -95,17 +91,27 @@ class LoopEnd : public LoopBase {
     void set_work_amount(size_t new_work_amount);
     void set_increment(size_t new_increment);
     void set_evaluate_once(bool once);
+    void set_work_with_buffer(bool buffer);
```

Comment:
`set_work_with_buffer` is not implemented and never used. remove?

Reply:
Yeap, it's redundant now. Thanks!